src/db_api_util.ts

```diff
@@ -1021,6 +1021,33 @@ export async function logVoterStatusToDb(
   }
 }
 
+export async function logInitialVoterStatusToDb(
+  userId: string,
+  userPhoneNumber: string,
+  twilioPhoneNumber: string
+): Promise<void> {
+  logger.info(`ENTERING DBAPIUTIL.logInitialVoterStatusToDb`);
+  const client = await pool.connect();
+  try {
+    const isDemo =
+      twilioPhoneNumber === process.env.DEMO_PHONE_NUMBER ? true : null;
+    await client.query(
+      `INSERT INTO voter_status_updates (user_id, user_phone_number, voter_status, is_demo)
+      SELECT $1, $2, 'UNKNOWN', $3
```

Comment:
postgres doesn't have a ``REPLACE INTO`` ala mysql; this is the way to do a conditional insert, so that we only insert UNKNOWN if the ALREADY_VOTED isn't already present.